Components/Queues/QueueMessage.php

```diff
@@ -132,4 +142,9 @@ public function getDestinationURI()
     {
         return $this->getHeader(Message::HEADER_FROM);
     }
+
+    public function getMessageId()
```

Comment:
Because we specify int in setMessage, I think we can add `int` as return typehinting in `getMessageId`

Reply:
Return type hint added.